app/Config/core.php

```diff
@@ -237,7 +237,7 @@
  * Set to `true` to apply timestamps when debug > 0. Set to 'force' to always enable
  * timestamping regardless of debug value.
  */
-	//Configure::write('Asset.timestamp', true);
+    //Configure::write('Asset.timestamp', true);
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Asset.timestamp', true);" but found "//Configure::write('Asset.timestamp', true);"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment